src/index.ts

```diff
@@ -0,0 +1,75 @@
+import {AddressBook} from "./identity/hcs/address-book";
+import {ArraysUtils} from "./utils/arrays-utils";
+import {CredentialSubject} from "./identity/hcs/vc/credential-subject";
+import {DidDocumentBase} from "./identity/did-document-base";
+import {DidDocumentJsonProperties} from "./identity/did-document-json-properties";
+import {DidMethodOperation} from "./identity/did-method-operation";
+import {DidParser} from "./identity/did-parser";
+import {DidSyntax} from "./identity/did-syntax";
+import {Hashing} from "./utils/hashing";
+import {HcsDidMessage} from "./identity/hcs/did/hcs-did-message";
+import {HcsDidResolver} from "./identity/hcs/did/hcs-did-resolver";
+import {HcsDidRootKey} from "./identity/hcs/did/hcs-did-root-key";
+import {HcsDidTopicListener} from "./identity/hcs/did/hcs-did-topic-listener";
+import {HcsDidTransaction} from "./identity/hcs/did/hcs-did-transaction";
+import {HcsDid} from "./identity/hcs/did/hcs-did";
+import {HcsIdentityNetworkBuilder} from "./identity/hcs/hcs-identity-network-builder";
+import {HcsIdentityNetwork} from "./identity/hcs/hcs-identity-network";
+import {HcsVcDocumentBase} from "./identity/hcs/vc/hcs-vc-document-base";
+import {HcsVcDocumentHashBase} from "./identity/hcs/vc/hcs-vc-document-hash-base";
+import {HcsVcDocumentJsonProperties} from "./identity/hcs/vc/hcs-vc-document-json-properties";
+import {HcsVcMessage} from "./identity/hcs/vc/hcs-vc-message";
+import {HcsVcOperation} from "./identity/hcs/vc/hcs-vc-operation";
+import {HcsVcStatusResolver} from "./identity/hcs/vc/hcs-vc-status-resolver";
+import {HcsVcTopicListener} from "./identity/hcs/vc/hcs-vc-topic-listener";
+import {HederaDid} from "./identity/hedera-did";
+import {JsonClass} from "./identity/hcs/json-class";
+import {MessageEnvelope} from "./identity/hcs/message-envelope";
+import {MessageListener} from "./identity/hcs/message-listener";
+import {MessageMode} from "./identity/hcs/message-mode";
+import {MessageResolver} from "./identity/hcs/message-resolver";
+import {MessageTransaction} from "./identity/hcs/message-transaction";
+import {Message} from "./identity/hcs/message";
+import {SerializableMirrorConsensusResponse} from "./identity/hcs/serializable-mirror-consensus-response";
+import {TimestampUtils} from "./utils/timestamp-utils";
+import {Validator} from "./utils/validator";
+import {Issuer} from "./identity/hcs/vc/issuer";
+
+export {
```

Comment:
Am I right in understanding that users of this library will have access to all exported APIs? Are all of these APIs necessary to export?

Reply:
I think yes becouse users will want to use type defenishions for example. It is possible to remove exports such low level classes as DidDocumentBase, but with it users can create their own documents.